dags/telemetry_aggregates_parquet.py

```diff
@@ -0,0 +1,28 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+
+default_args = {
+    'owner': 'robhudson@mozilla.com',
+    'depends_on_past': True,
+    'start_date': datetime(2018, 9, 21),
+    'email': ['telemetry-alerts@mozilla.com', 'robhudson@mozilla.com', 'frank@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 3,
+    'retry_delay': timedelta(minutes=30),
+}
+
+dag = DAG('telemetry_aggregates_parquet', default_args=default_args, schedule_interval='@daily')
+
+telemetry_aggregate_parquet_view = EMRSparkOperator(
```

Comment:
Want to run this on databricks?

Reply:
I'm not sure? Should we? How does one do that?

Reply:
The Longitudinal job does that, check how it's done there.

Reply:
Cool, I'll check it out and switch to that.

Reply:
Only jobs that run via `mozetl_envvar` and `tbv_envvar` and the corresponding runners are supported via Databricks now. Custom scripts won't work without modifying the operator. Sorry for the lack of documentation on this behavior, it should be a bit more explicit.